knowledge-base/kb-changelocalization-at-runtime.md

```diff
@@ -0,0 +1,42 @@
+---
+title: PDFViewer Localization of The FindDialog During Runtime
```

Comment:
Shouldn't it be "Change Localization of The FindDialog of PdfViewer During Runtime"? Currently, it does not make much sense with this wording.